packages-presets/transformer-directives/src/apply.ts

```diff
@@ -119,6 +127,7 @@ export async function parseApply({ code, uno, applyVariable }: TransformerDirect
         code.appendRight(childNode!.loc!.end.offset + semicolonOffset, body)
     }
   }
+  // todo: after transformered remove empty css like `selector{}`
```

Comment:
Do you want to continue completing this TODO in this PR?

Reply:
This function has nothing to do with this pr. I just think it is a point that can be optimized and can be put in the later pr.